lang/c/c.talon

```diff
@@ -145,14 +145,15 @@ push brackets:
     insert("{variable} {letter} ")
 
 # Ex. int * testFunction
-fun <user.function> <phrase>:
-    insert("{function} ")
-    insert(user.formatted_text(phrase, "PRIVATE_CAMEL_CASE,NO_SPACES"))
-    insert("()")
-    edit.left()
+# TODO: these clearly don't want to use function_key, so what do they want to use?
+# fun <user.function> <phrase>:
+#     insert("{function} ")
+#     insert(user.formatted_text(phrase, "PRIVATE_CAMEL_CASE,NO_SPACES"))
+#     insert("()")
+#     edit.left()
 
-<user.function>:
-    insert("{function} ")
+# <user.function>:
```

Comment:
This was probably supposed to be `c_functions`. 

This is probably be out of scope for this, but I think this should probably be moved over to use the `code_functions` and `code_libraries`, as done for python and R. And maybe move to the new code_insert_function stuff, which I think can basically be copied from csharp while we're here, or open an item for it. This way, we benefit from the imgui and stuff in code.py. I guess we'd need to explore how to do that for the libraries though.

https://github.com/knausj85/knausj_talon/blob/0a9c40f9e979ba7b1bae24b94fd57635914ee613/lang/csharp